### PR TITLE
add in-container vendor

### PR DESCRIPTION
**Pull request by SamuelStuchly**
Add option to run make vendor in container like other targets, to avoid issues with go module caching depending on what you already have on your machine.

### CODE REVIEW

**Review by elmiko — APPROVED** [review]
/approve

File: hack/go-mod.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+go mod tidy
+go mod vendor
+go mod verify
```
Comment (JoelSpeed) [inline hack/go-mod.sh:4]: Nit: Should have a new line at the end of this file ideally

**Pull request merged**